Dockerfile.alpine

```diff
@@ -0,0 +1,121 @@
+#
```

Comment:
Comparing the new Dockerfiles to https://github.com/sonatype/docker-nexus-iq-server/blob/master/Dockerfile is the easiest way to see the diffs. I use: https://www.diffchecker.com/